test/OpenTelemetry.Tests/Logs/BatchExportLogRecordProcessorTest.cs

```diff
@@ -0,0 +1,205 @@
+// <copyright file="BatchExportLogRecordProcessorTest.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+#if !NET452 && !NET46
+#if NETCOREAPP2_1
+using Microsoft.Extensions.DependencyInjection;
+#endif
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using Microsoft.Extensions.Logging;
+using OpenTelemetry.Exporter;
+using OpenTelemetry.Logs;
+using Xunit;
+
+namespace OpenTelemetry.Tests.Logs
+{
+    public class BatchExportLogRecordProcessorTest
```

Comment:
Both Simple and Batch export processor are generic - we dont need to test it separately here. it should be tested outside of Logs.

Reply:
Removed Simple and BatchExport Processor tests for Logs.